kong/key.go

```diff
@@ -0,0 +1,22 @@
+package kong
+
+// Key represents a JWK or PEM key in Kong.
+// +k8s:deepcopy-gen=true
+type Key struct {
+	ID        *string   `json:"id,omitempty" yaml:"id,omitempty"`
+	CreatedAt *int64    `json:"created_at,omitempty" yaml:"created_at,omitempty"`
+	UpdatedAt *int64    `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
+	Set       *KeySet   `json:"set,omitempty" yaml:"set,omitempty"`
+	Name      *string   `json:"name,omitempty" yaml:"name,omitempty"`
+	Kid       *string   `json:"kid,omitempty" yaml:"kid,omitempty"`
+	Jwk       *string   `json:"jwk,omitempty" yaml:"jwk,omitempty"`
+	Pem       *PEM      `json:"pem,omitempty" yaml:"pem,omitempty"`
```

Comment:
Other types in this lib keep capitalization for acronym fields, so this one should as well:

```suggestion
	KID       *string   `json:"kid,omitempty" yaml:"kid,omitempty"`
	JWK       *string   `json:"jwk,omitempty" yaml:"jwk,omitempty"`
	PEM       *PEM      `json:"pem,omitempty" yaml:"pem,omitempty"`
```

Reply:
done